scripts/enforce_imports.py

```diff
@@ -143,23 +144,23 @@ def _is_correct_order(self,
                           node2: ast.AST,
                           order_info1: ModuleOrderInfo,
                           order_info2: ModuleOrderInfo):
-        return (node1.lineno1 < node2.lineno2) == (order_info1 < order_info2)
+        return (node1.lineno < node2.lineno) == (order_info1 < order_info2)
 
     def _order_info(self, node: ast.AST) -> Optional[ModuleOrderInfo]:
         """
         >>> visitor = ImportVisitor(lines=[])
-        >>> visitor._order_info(ast.AST(names=[ast.AST(name='azul.indexer'), ast.AST(name='azul.service')]))
+        >>> visitor._order_info(one(ast.parse('import azul.indexer, azul.service').body))
 
-        >>> visitor._order_info(ast.AST(names=[ast.AST(name='azul.indexer')]))
+        >>> visitor._order_info(one(ast.parse('import azul.indexer').body))
         (<ModuleType.internal: 3>, 'azul.indexer', '')
 
-        >>> visitor._order_info(ast.AST(names=[ast.AST(name='BaseIndexer')], module='azul.indexer'))
+        >>> visitor._order_info(one(ast.parse('from azul.indexer import BaseIndexer').body))
         (<ModuleType.internal: 3>, 'azul.indexer', 'BaseIndexer')
 
-        >>> visitor._order_info(ast.AST(names=[ast.AST(name='one')], module='more_itertools'))
+        >>> visitor._order_info(one(ast.parse('from more_itertools import one').body))
         (<ModuleType.external_dependency: 2>, 'more_itertools', 'one')
 
-        >>> visitor._order_info(ast.AST(names=[ast.AST(name='itertools')]))
+        >>> visitor._order_info(one(ast.parse('import itertools').body))
         (<ModuleType.python_runtime: 1>, 'itertools', '')
         """
         try:
```

Comment:
Kind of a nit, but consider using a conditional here instead of `try:... one(...)... except ...`, It's less lines and I think more direct.